scout/constants/variant_tags.py

```diff
@@ -37,6 +39,9 @@
     "category",
     "dismiss_variant",
     "variant_rank",
+    "sub_category",
+    "end_chrom",
```

Comment:
👏🏻

Reply:
![Screenshot 2022-04-01 at 12 26 08](https://user-images.githubusercontent.com/758570/161245940-22600aa3-b6bb-4ec2-83a6-58f93be6df95.png)

Reply:
Well done, let's merge before we see something else?